Doc/Manual/Python.html

```diff
@@ -4134,8 +4134,9 @@ <H3><a name="Python_stable_abi">33.6.6 Stable ABI</a></H3>
 
 <p>
 The default SWIG command line options generate code that enable the limited API/stable ABI.
-Some options, such as <tt>-builtin</tt>, <tt>-fast</tt> (used by <tt>-O</tt>) do not use the limited API and hence when <tt>Py_LIMITED_API</tt> is defined there will be missing symbols during compilation.
+Some options, such as <tt>-builtin</tt>, <tt>-fastproxy</tt> (used by <tt>-O</tt>) do not use the limited API and hence when <tt>Py_LIMITED_API</tt> is defined there will be missing symbols during compilation.
 Compatibility of user's custom typemaps is of course dependent on the Python APIs used in the typemaps.
+Use of typemaps requires passing the <tt>-fastdispatch</tt> option to SWIG (which is typically otherwise enabled by the abi3-incompatible <tt>-O</tt>).
```

Comment:
If you provide your own typemaps then it is up to you to ensure it works with or without `-fastdispatch`, so this statement is wrong.

I'm not sure what your problem is given the information in this issue. Please provide a small cutdown testcase if you need help writing a typemap.

Reply:
> If you provide your own typemaps then it is up to you to ensure it works with or without -fastdispatch, so this statement is wrong.

The problematic typemap is [very simple](https://github.com/openmeeg/openmeeg/blob/4cf760b23f88ddb0523b66235e01debe2795bd42/wrapping/python/openmeeg/_openmeeg.i#L357-L362):
```
    %typemap(in) Vector& {
        $1 = new_OpenMEEG_Vector($input);
    }
```
Which just calls [this little helper](https://github.com/openmeeg/openmeeg/blob/4cf760b23f88ddb0523b66235e01debe2795bd42/wrapping/python/openmeeg/_openmeeg.i#L201-L218). Would you expect this to work?

I could try to trim the example but I am far from a `swig` expert. Do you happen to have some tiny sample project I could start from and build out? Going the opposite direction starting from OpenMEEG and cutting down would be a nightmare :sweat_smile:

Reply:
The SWIG developers do not know OpenMEEG, you really ought to ask them for help.

Reply:
Sorry I thought that would be simple enough to have a sense of whether this could be a SWIG problem rather than an OpenMEEG one. Any thoughts/ideas on my second question regarding:

> Do you happen to have some tiny sample project I could start from and build out?

Reply:
I'm not sure what `Vector` is but if it is anything like `std::vector`, then simply start with:
```
%include <std_vector.i>
%template(VectorInt) std::vector<int>;
```